vts/trustedservices/trustedservices_grpc.go

```diff
@@ -360,38 +360,44 @@ func (o *GRPC) GetAttestation(
 		return o.finalize(appraisal, err)
 	}
 
-	appraisal.EvidenceContext.ReferenceId = extracted.ReferenceID
+	appraisal.EvidenceContext.ReferenceIds = extracted.ReferenceIDs
 
 	o.logger.Debugw("constructed evidence context",
-		"software-id", appraisal.EvidenceContext.ReferenceId,
-		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorId)
+		"software-id", appraisal.EvidenceContext.ReferenceIds,
+		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorIds)
 
-	endorsements, err := o.EnStore.Get(appraisal.EvidenceContext.ReferenceId)
-	if err != nil && !errors.Is(err, kvstore.ErrKeyNotFound) {
-		return o.finalize(appraisal, err)
-	}
+	var tendorsements []string
```

Comment:
curiosity: what is the `t` in `tendorsements` exactly supposed to mean? 😄

Reply:
I meant totalEndorsements via `tendorsements` but may be it is not very helpful! 
I will rename it!

Reply:
ah, thanks!  Maybe we can use `endorsement` for each item and `endorsements` for the slice?

Reply:
in the inner block also, we return `endorsements` Hence I used slightly longer but `multEndorsements`